src/adtechmedia-config.php

```diff
@@ -45,6 +59,7 @@ public static function get( $name ) {
 	public static function set_api_end_point() {
 		if ( self::is_localhost() ) {
 			self::$conf['api_end_point'] = 'https://api-dev.adtechmedia.io/dev/';
+			self::$conf['sw_js_url']     = 'https://adm-dev.adtechmedia.io/atm-admin/atm-build/sw.js';
```

Comment:
Please define dev endpoints in config and change it to `https://api-dev.adtechmedia.io/atm-admin/atm-build/sw.js`